### PR TITLE
Fix/topbar styles

### PR DESCRIPTION
**Pull request by michalisKout**
## Description

This PR changes some of the styles of the top bar due to the recent changes in the designs: [here](https://www.figma.com/file/8kMPBNYHHXz2AtkzeeDmk5/Design-System?node-id=1125%3A1564)

## Screenshot

Screenshot 2021-01-08 at 2 45 16 PM

## Test Plan

snapshot testing

### CODE REVIEW

File: src/components/TopAppBar/components/Logo.style.ts [hunk 1]
```diff
@@ -5,11 +5,9 @@ import { flexCenter } from '../../../theme/functions';
 const wrapper = (theme: Theme): SerializedStyles => css`
   ${flexCenter};
   background-color: transparent;
-  margin: ${theme.spacing.md} ${theme.spacing.md} ${theme.spacing.md} 0px;
+  margin: ${theme.spacing.md} 72px ${theme.spacing.md} ${theme.spacing.md};
```
Comment (Christos-Zacharopoulos) [inline src/components/TopAppBar/components/Logo.style.ts:8]: This is outside the scope of this PR. 

Maybe we need more sizes in spacing to accommodate this and everything else that might use it.

**Review by Christos-Zacharopoulos — APPROVED** [review]

File: src/components/TopAppBar/components/Logo.style.ts (hunk 1 above)
Comment (michalisKout) [reply src/components/TopAppBar/components/Logo.style.ts:8]: I agree this should be contacted with the design team and add a new larger spacing to cover this and other related cases.

**Comment by kouloumos** [comment]
@michalisKout there are additional changes on the topbar as shown in the [CMO designs](https://www.figma.com/file/bd3uFRMADWac01IqLhDvHN/CMO-Portal?node-id=1%3A7)

**Comment by michalisKout** [comment]
> @michalisKout there are additional changes on the topbar as shown in the [CMO designs](https://www.figma.com/file/bd3uFRMADWac01IqLhDvHN/CMO-Portal?node-id=1%3A7)

@kouloumos I believe these changes are project-specific. As discussed with @panvourtsis, all DS related projects should follow the DS designs in terms of all shared components.

**Comment by panvourtsis** [comment]
one quick thing that we missed is that the search should be hidden if no `onSearchHandler` exists

**Review by panvourtsis — COMMENTED** [review]
LGTM

**Review by panvourtsis — APPROVED** [review]
